source/content/guides/wordpress-composer/03-behat-testing-composer.md

```diff
@@ -0,0 +1,41 @@
+---
+title: WordPress with Composer on Pantheon
+subtitle: Use Behat Testing with Your Composer-managed WordPress Site 
+description: Learn how to use Behat testing with your Composer-based WordPress site. 
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/behat-testing-composer
+anchorid: behat-testing-composer
+---
+
+This section provides information on how to use Behat testing with your Composer-based WordPress site.
+
+Behat testing uses `.ci/test/behat` and `tests/behat`. [Behat](https://behat.org/en/latest/) is an acceptance/end-to-end testing framework written in PHP. It facilitates testing the fully-built WordPress site on Pantheon. [WordHat](https://wordhat.info/) is used to help integrate Behat and WordPress.
```

Comment:
It looks like you're referencing files that already exist (`.ci/test/behat`), but I'm unsure where these files would come from. Could you advise @whitneymeredith?